scripts/boxesserver

```diff
@@ -242,10 +242,10 @@ class BServer:
             if len(group._group_actions) == 1 and isinstance(group._group_actions[0], argparse._HelpAction):
                 continue
             prefix = getattr(group, "prefix", None)
-            result.append('''<h3 id="h-%s" class="open" onclick="showHide(%s)">%s</h3>\n<table id="%s">\n''' % (groupid, groupid, _(group.title), groupid))
+            result.append('''<h3 id="h-%s" class="open" onclick="showHide(%s)">%s</h3>\n<table role="presentation" id="%s">\n''' % (groupid, groupid, _(group.title), groupid))
```

Comment:
Adds presentation view to layout table. Probably not the best fix as according to[ W3 WAI best practice is using CSS for visual styling](https://www.w3.org/WAI/tutorials/tables/), but gets it more screen reader friendly without requiring a large rewrite of the web ui.